geopyspark/geotrellis/tms.py

```diff
@@ -16,8 +16,8 @@ class TileRender(object):
     rendering functions.
 
     Args:
-        render_function (numpy.ndarray => bytes): A function to convert a numpy
-            array to a collection of bytes giving a binary image file.
+        render_function (Tile => bytes): A function to convert geopyspark.geotrellis.Tile
```

Comment:
Just noticing this docstring is incorrect.  The render function should produce an Image, not bytes.